stack/stack_mac_m1_gnu.yaml

```diff
@@ -0,0 +1,359 @@
+gnu:
```

Comment:
Same for this

Reply:
This is only for GNU collection, when it is installed in advance of the build. MPI could be installed during the hpc-stack build.